viadot/sources/cloud_for_customers.py

```diff
@@ -5,56 +5,169 @@
 from urllib.parse import urljoin
 from ..config import local_config
 
+import time
+from requests.adapters import HTTPAdapter
+from requests.exceptions import ConnectionError, HTTPError, ReadTimeout, Timeout
+from urllib3.exceptions import ProtocolError
+import logging
+import re
+from requests.packages.urllib3.util.retry import Retry
 
-class CloudForCustomers(Source):
-    """
-    Fetches data from Cloud for Customer.
+url = "https://my341115.crm.ondemand.com/sap/c4c/odata/ana_businessanalytics_analytics.svc/$metadata?entityset=RPZ1F90A299C830A29C6659D3QueryResult"
+
+
+def column_mapper(url=url):
+    credentials = local_config.get("CLOUD_FOR_CUSTOMERS")
```

Comment:
`credentials` should be a parameter to the function, not taken from the global config variable. Also, since the base `Source` class provides a `credentials` parameter, user should be able to pass them like this:
```python
my_source = CloudForCustomers(credentials={"username": "admin", "password": "1234"})
```